### PR TITLE
Update services.json

### PR DESCRIPTION
**Pull request by rpslack**
AfreecaTV will be adding an "RTMP Stream Address" section. As we offer AfreecaTV in multiple languages, we have added a separate server.

### CODE REVIEW

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -710,5 +710,89 @@
...
+            "recommended": {
+                "keyint": 1,
+                "profile": "main",
+                "max video bitrate": 5000,
+                "max audio bitrate": 192
+            }
+        }
+        {
+            "name": "アフリカTV",
+            "common" : true,
```
Comment (jumoog) [inline plugins/rtmp-services/data/services.json]: please remove common see -> https://github.com/jp9000/obs-studio/pull/577#issuecomment-234386514

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -710,5 +710,89 @@
                 "max video bitrate": 3500
             }
         }
+        {
+            "name": "Afreeca.TV",
+            "common" : true,
```
Comment (jumoog) [inline plugins/rtmp-services/data/services.json]: please remove common see -> https://github.com/jp9000/obs-studio/pull/577#issuecomment-234386514

File: plugins/rtmp-services/data/services.json [hunk 3]
```diff
@@ -710,5 +710,89 @@
...
+            "recommended": {
+                "keyint": 1,
+                "profile": "main",
+                "max video bitrate": 5000,
+                "max audio bitrate": 192
+            }
+        }
+        {
+            "name": "艾菲卡TV",
+            "common" : true,
```
Comment (jumoog) [inline plugins/rtmp-services/data/services.json]: please remove common see -> https://github.com/jp9000/obs-studio/pull/577#issuecomment-234386514

File: plugins/rtmp-services/data/services.json [hunk 4]
```diff
@@ -710,5 +710,89 @@
...
+             "recommended": {
+                "keyint": 1,
+                "profile": "main",
+                "max video bitrate": 5000,
+                "max audio bitrate": 192
+            }
+        }
+        {
+            "name": "아프리카TV",
+            "common" : true,
```
Comment (jumoog) [inline plugins/rtmp-services/data/services.json]: please remove common see -> https://github.com/jp9000/obs-studio/pull/577#issuecomment-234386514

**Comment by Gol-D-Ace** [comment]
Wrong commit title. (rtmp-services: Add AfreecaTV streaming service)
Wrong ingest urls. (missing rtmp://)
Isn't valid json.
Should not be common.
Missing version increment in package.json

Please fix this issues if you want to get this merged.

**Comment by rpslack** [comment]
I fixed issues. please check again.

**Comment by Gol-D-Ace** [comment]
You're still missing the version increment in the package.json and also the commit title is still wrong.
Also you should squash your commits into one.

**Pull request closed without merging**